README.md

```diff
@@ -52,6 +52,8 @@ Python 3.6 or later, CMake 3.4.3 or later, and openssl (1.0) are also required.
     apt install build-essential llvm clang libclang-dev cmake libssl-dev pkg-config python3
     ```
 
+Depending on the LLVM distribution, the `llvm-dev` package may also be required.
+
```

Comment:
```suggestion
For example, the official LLVM packages from [apt.llvm.org](https://apt.llvm.org/) require `llvm-dev` to be installed.

```

Reply:
ok. I'll check which ones, and follow up (probably tomorrow), because not all of them may have this structure.

Reply:
Okay, thanks!  If we can figure out all of them that would be great, but I think [apt.llvm.org](https://apt.llvm.org/) specifically should be mentioned as that's a very common way to install LLVM.

Reply:
I've checked the x64 packages for Ubuntu focal on the repository, and in all the versions (9/14), the config file was always in `llvm-dev`, so the commit message is good as suggested! :smile: